SUSE/README.md

```diff
@@ -3,3 +3,7 @@
 - [SUSE](https://www.suse.com) is implementing SigStore methods in its Container and Product deliveries.
 - The [SUSE Container Registry](https://registry.suse.com/) is now also signing the SUSE containers using "cosign" (additionaly to Notary and Atomic methods) and also uploading signatures to the Linux Foundation Transparency Rekor log.
```

Comment:
nitpicking, using the same naming conventions adopted by the sigstore project. Plus addressing a small typo:

```suggestion
- The [SUSE Container Registry](https://registry.suse.com/) is now also signing the SUSE containers using Cosign (additionally to Notary and Atomic methods) and also uploading signatures to the Linux Foundation Transparency Rekor log.
```